benchmarks/Transformers/BM25TransformerBench.php

```diff
@@ -14,6 +14,16 @@ class BM25TransformerBench
 {
     protected const NUM_SAMPLES = 10000;
 
+    /**
+     * @var \Rubix\ML\Datasets\Unlabeled
+     */
+    public $dataset;
```

Comment:
Does this need to be a public property?

Reply:
no, good point. I'll make these consistently `protected` 👍